### DISCUSSION TITLE
How do you update Git LFS

### DISCUSSION THREAD

**Discussion by bronsiOaG** [git-lfs/git-lfs#4371] [Q&A]
I need to install the security update, but can't figure out how that's done. 
I tried it via the downloaded installer, but ">git lfs version" still shows I'm using an old version. 

Thanks.

**Comment by chrisd8088** [comment]


**Reply by v-chilso** [reply]
On Windows using Git Bash I found this helpful post
https://stackoverflow.com/questions/13790592/how-to-upgrade-git-on-windows-to-the-latest-version

The command used is
git update-git-for-windows
